src/containers/BooksForm.js

```diff
@@ -35,15 +35,18 @@ class BooksForm extends Component {
     addBook(this.state);
     this.setState(state => ({
       id: state.id + 1,
+      title: '',
+      category: categories[0].category,
     }));
   }
 
   render() {
+    const { title, category } = this.state;
     const options = categories.map(element => <option key={element.id}>{element.category}</option>);
     return (
       <form onSubmit={this.handleSubmit}>
-        <input type="text" name="title" placeholder="title" onChange={this.handleChange} required />
-        <select name="category" onChange={this.handleChange}>
+        <input type="text" name="title" value={title} placeholder="title" onChange={this.handleChange} required />
+        <select name="category" value={category} onChange={this.handleChange}>
           {options}
         </select>
         <button type="submit" onSubmit={this.handleSubmit}>Submit</button>
```

Comment:
We can delete the additional `onSubmit` in the `button` element.
```suggestion
        <button type="submit" >Submit</button>
```